clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/MoveExecutor.scala

```diff
@@ -177,10 +177,11 @@ class MoveExecutor[TI <: TransferIndex](moveCommand: MoveCommand[TI])
           .sequence(googleDeletes)
           .recover {
             case ex =>
+              val notDeleted = oldPaths.par.filter(ioUtil.googleObjectExists).seq
```

Comment:
We shouldn't need to hit the network again, we can already get the info of what failed by being smarter about how we collect all the delete `Future`s.

Reply:
I'll give that a go. Probably have to transform to a sequence of Eithers.